core-bundle/src/Util/PackageUtil.php

```diff
@@ -21,16 +21,21 @@ class PackageUtil
      */
     public static function getVersion(string $packageName): string
     {
-        $version = Versions::getVersion($packageName);
+        $version = InstalledVersions::getPrettyVersion($packageName) ?? '';
```

Comment:
That's incorrect. You should simply use `getPrettyVersion()` and not modify at all here. This is the one you would want to use when showing a version to the user.

Reply:
That's wrong, because `getPrettyVersion()` can return `null` and `getVersion()` cannot. We could use a string cast instead of `?? ''` although the result will be the same. 🤷‍♂️

Reply:
Want to explain to me how Composer works? :) Yes, the pretty version is optional, we can fall back to the normalized one if there is none. But we always need 2 methods and not just one because the normalized and pretty versions are not the same. Right now you deprecated the normalized one which is almost always the one you want to use.

Reply:
There are just two representations. One of which is meant to be used for the users to improve readability and the other one is the one used for comparisons. What third options do we need then?

Reply:
> Want to explain to me how Composer works? :)

No, but obviously I have to explain to you how the PackageUtil class works. We only need two methods in the future: `getVersion()` and `getContaoVersion()`. Both are supposed to return the pretty version. For anything beyond that, please directly use the InstalledVersions class.